msr_batch.c

```diff
@@ -1,4 +1,4 @@
-// Copyright 2011-2021 Lawrence Livermore National Security, LLC and other
+// Copyright 2012-2021 Lawrence Livermore National Security, LLC and other
```

Comment:
Start copyright year at 2011

Reply:
I'm not sure if this file existed in 2011.  Github history doesn't go back that far and.... never mind.  2011 it is.